### PR TITLE
Random card logo

### PR DESCRIPTION
**Pull request by Caroline-CD**

### CODE REVIEW

**Review by John-athan14 — DISMISSED** [review]

File: src/Containers/CardDescription.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+.Container {
```
Comment (Hacene010) [inline src/Containers/CardDescription.css]: Use camelCase for all your calsseName to be coherent plz

File: src/Containers/CardDescription.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+.CardOracle {
+  color: white;
+  font-size: 15px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.frame {
```
Comment (Hacene010) [inline src/Containers/CardDescription.css:60]: same here !!

File: src/Containers/CardDescription.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  width: 30vw;
+  background-color: white;
+  opacity: 0.3;
+}
+
+.logo-reload {
```
Comment (Hacene010) [inline src/Containers/CardDescription.css]: Replace the kebab-case by camelCase too.

File: src/Containers/CardDescription.jsx [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+  const [card, setCard] = useState(sampleCard);
+
+  const GetRandomCards = () => {
+    axios.get(`https://api.scryfall.com/cards/random`).then(({ data }) => {
+      setCard({ ...data });
+    });
+  };
+
+  return (
+    <div className="Container">
```
Comment (Hacene010) [inline src/Containers/CardDescription.jsx]: Same problem with camelCase : /

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Use camelCase everywhere plz to be coherent

**Review by John-athan14 — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]
that's better

**Pull request merged**